pyanaconda/core/kernel.py

```diff
@@ -22,12 +22,15 @@
 from pyanaconda.core.constants import CMDLINE_APPEND, CMDLINE_LIST, CMDLINE_FILES
 
 
-class KernelArguments(OrderedDict):
+class KernelArguments():
```

Comment:
We don't have tests for this class. Please, write some or create an issue for that.